piecrust-uplink/src/state.rs

```diff
@@ -268,10 +268,10 @@ pub fn height() -> u64 {
 }
 
 /// Return the current module's owner.
```

Comment:
Shouldn't we mention the reasoning here for why we allow N size and why it is even possible to do so in Dusk modules? Given variable size/self defined keys are very uncommon within this context in other blockchains.

Reply:
We actually technically are not allowing "variable size" keys at all. We're just allowing the consumer of `piecrust` to define the size of those keys themselves, and possibly create a wrapper around `piecrust_uplink::owner()` returning whatever key type they might want to use. I'm also trying very hard to have piecrust not use cryptography whatsoever - in fact even the concept of "keys" is not present here.

Downstream, in `rusk_abi`, we should provide a function wrapping around the uplink's `owner` function, returning something like a `dusk-bls12_381_sign::PublicKey`, and we would guarantee in `rusk` that this is always set to a valid key.

Reply:
> 

Ah yes. I get what you mean now. Resolving this.